Assets/Scripts/PlayerControl/PlayerController.cs

```diff
@@ -6,14 +6,30 @@
 public class PlayerController : MonoBehaviour
 {
     [Header("Movement Params")]
-    [SerializeField] float playerSpeed;
-    [SerializeField] float sensitivity;
+    [SerializeField] 
+    private float playerSpeed;
```

Comment:
I see you just added player speed here. It might be set way too high. Also noticed extreme gravity as well.